bee-network/examples/common/mod.rs

```diff
@@ -0,0 +1,6 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(dead_code)]
```

Comment:
Should we allow dead code?

Reply:
`examples` and `tests` are not production code, hence we can be less strict here IMHO. If you do not agree then I can change it oc.